packages/babel-plugin-minify-mangle-names/README.md

```diff
@@ -71,3 +71,4 @@ require("babel-core").transform("code", {
 + `blacklist` - A plain JS Object with keys as identifier names and values indicating whether to exclude
 + `eval` - mangle identifiers in scopes accessible by eval
 + `keepFnName` - prevent mangler from altering function names. Useful for code depending on `fn.name`
++ `topLevel` - mangle topLevel Identifiers
```

Comment:
I follow your reasoning for this mangling plugin, but it does worry my for the DCE plugin, since `topLevel` doesn't have much meaning in that case.